### PR TITLE
Tweak Smesher setup screen

### PR DESCRIPTION
**Pull request by brusherru**
This PR contains little tweaks that were discussed with @pigmej

### CODE REVIEW

**Comment by pigmej** [comment]
@brusherru 

gb/s -> GiB/s different unit :) Also not GIGABITS but GIBIBITS 
not TB but TiB etc.

File: app/components/node/PoSProfiler.tsx [hunk 1]
```diff
@@ -239,26 +255,45 @@ const PoSProfiler = ({ nextAction }: Props) => {
...
+                text="Speed of reading PoS and finding nonces in gigabits per second"
+              />
             </TCol>
             <TCol width={COL_WIDTH.SIZE}>
               Recommended PoS size
-              <Tooltip width={250} marginTop={0} text="TODO" />
+              <Tooltip
+                width={250}
+                marginTop={0}
+                text="Allocate not more than this amount to be sure that your machine able to produce the proof in time."
```
Comment (monikasmolarek) [inline app/components/node/PoSProfiler.tsx]: Allocate not more than this amount to be sure that your machine `is` able to produce the proof in time.

**Comment by monikasmolarek** [comment]
Testing on Linux Fedora, but afair the same behavior was also on other OS: 
once I run one benchmark, the next on the list gets the status "Running" but is never actually run. 
image

**Comment by brusherru** [comment]
@monikasmolarek @pigmej all fixed! Thanks :)

**Pull request merged**